evalml/tests/data_checks_tests/test_datetime_format_data_check.py

```diff
@@ -541,3 +515,287 @@ def test_datetime_many_duplicates_and_nans():
     result = dc.validate(X, y)
 
     assert result[2]["code"] == "DATETIME_NO_FREQUENCY_INFERRED"
+
+
+@pytest.mark.parametrize("input_type", ["pd", "ww"])
+@pytest.mark.parametrize(
+    "issue",
+    ["redundant", "missing", "uneven", "type_errors", None],
+)
+@pytest.mark.parametrize("datetime_loc", [1, "X_index"])
+def test_datetime_format_data_check_typeerror_uneven_intervals_multiseries(
```

Comment:
This is a long test with a lot of repeated code with the single series version of this test. Could you combine this with the pre-existing one and parametrize? You should be able to abstract out the `repeat` calls to generate multiple series by using `pd.Series.repeat(1)` for the single series case.